### ISSUE TITLE
Process aborts in the preparation stage

### ISSUE THREAD

**Issue by FranSarr** [plesk/centos2alma#176]
I attache the feedback file for your reference
[centos2alma_feedback.zip](https://github.com/plesk/centos2alma/files/14313472/centos2alma_feedback.zip)

**Describe the bug**

**Feedback archive**
*Please attach a feedback archive to the bug report.*
You could create it by calling `centos2alma --prepare-feedback`.
The archive will help us investigate the problem better and faster.

**Comment by Bitpalast** [comment]
@FranSarr Are you using Postfix on your server or what other email configuration are you using?

**Comment by Bitpalast** [comment]
Yes it does (but shouldn't). I was asking, because on the other report https://github.com/plesk/centos2alma/issues/177 the same was reported but with a different motivation to not having Postfix on the server. So it seems that when Postfix is missing, conversion fails.

**Comment by FranSarr** [comment]
Nice to know, thanks for your support and interest, pls let me know if I can be of any further help

Francisco Sarrias
Telecom Engineer
E: ***@***.***
P: +34 607 287953
W:https://www.linkedin.com/in/fsarrias/

**Comment by SandakovMM** [comment]
The problem should be fixed with #180 merged.
Please find attached centso2alma with the fix:  [centos2alma.zip](https://github.com/plesk/centos2alma/files/14373343/centos2alma.zip)

**Comment by FranSarr** [comment]
ThanksI will let you know my experience when I have time to deploy itBestFrancisco SarriasEl 22 feb 2024, a las 12:46, SandakovMM ***@***.***> escribió:﻿
Closed #176 as completed.

**Comment by FranSarr** [comment]
Hello
Sorry for my late reply, just to let you know that I tried with the latest version the upgrade and it worked like a charm, thanks for your work
Best regards


Francisco Sarrias
Telecom Engineer
E: ***@***.***
P: +34 607 287953
W:https://www.linkedin.com/in/fsarrias/
